### DISCUSSION TITLE
How to create message command only for direct message?

### DISCUSSION THREAD

**Discussion by bezvozni** [nextcord/nextcord#488] [Q&A] [answered]
I want to add a contextual command to messages in a personal conversation with a bot. That would be nowhere else, except for the personal correspondence of this team was not visible. Is this possible and hint how to do it, if yes?

**Comment by ooliver1** [comment, marked as answer]
Discord has not provided much restrictive functionality, including guild/dm only and permissions. 

All you can do right now is local validation but this won't hide/disable commands unavailable to the user sadly :(

**Comment by Alex-L3** [comment]
Came across this post from a search with the same task.
perhaps the author meant a simple command (which he wants to send in dm>bot to perform some action. if so, the solution seems to be simple

```
from nextcord.ext import commands
bot = commands.Bot(command_prefix='!')

discord_token = 'INSERT_TOKEN' #STRING_VALUE
your_channel_id = 123456789 #INTEGER_VALUE

@bot.command()
async def your_command_name(ctx):
    channel = bot.get_channel(your_channel_id)
    await channel.send('sssssoooooommmeeee_test_message')

if __name__ == '__main__':
    bot.run(discord_token)
```
if a bot in dm receives a !your_command_name command, then it will send a message to a specific channel.
console gives a warning bot.py:187: MissingMessageContentIntentWarning, but its working
